invenio_communities/assets/semantic-ui/js/invenio_communities/requests/requests.js

```diff
@@ -0,0 +1,536 @@
+        </Button>
+      </Button.Group>
+    );
+  }
+}
+
+RequestStatusFilterComponent.propTypes = {
+  updateQueryState: PropTypes.func.isRequired,
+  currentQueryState: PropTypes.object.isRequired,
+};
+
+export const RequestStatusFilter = withState(RequestStatusFilterComponent);
+
+export const CommunityRequestsSearchLayout = (props) => {
+  return (
+    <Container>
+      <Grid>
+        <Grid.Row columns={3}>
+          <Grid.Column width={4} />
+          <Grid.Column width={3}>
+            <RequestStatusFilter />
+          </Grid.Column>
+          <Grid.Column width={9}>
+            <SearchBar placeholder={i18next.t("Search requests...")} />
+          </Grid.Column>
+        </Grid.Row>
+        <Grid.Row>
+          <Grid.Column width={4}>
+            <SearchAppFacets aggs={props.config.aggs} />
+          </Grid.Column>
+          <Grid.Column width={12}>
+            <SearchAppResultsPane layoutOptions={props.config.layoutOptions} />
+          </Grid.Column>
+        </Grid.Row>
+      </Grid>
+    </Container>
+  );
+};
+
+export const RequestsFacets = ({ aggs, currentResultsState }) => {
+  return (
+    <aside aria-label={i18next.t("filters")} id="search-filters">
+      {aggs.map((agg) => {
+        return (
+          <div className="ui accordion" key={agg.title}>
+            <BucketAggregation title={agg.title} agg={agg} />
+          </div>
+        );
+      })}
+      <SearchHelpLinks />
+    </aside>
+  );
+};
+
+export const CommunityRequestsEmptyResults = (props) => {
+  const { queryString, userSelectionFilters } = props;
+  const is_open = userSelectionFilters.some(
+    (obj) => obj.includes("is_open") && obj.includes("true")
+  );
+  const filtersToNotReset = userSelectionFilters.find((obj) =>
+    obj.includes("is_open")
+  );
+  const elementsToReset = {
+    queryString: "",
+    page: 1,
+    filters: [filtersToNotReset],
+  };
+
+  const AllDone = () => {
+    return (
+      <>
+        <Header as="h1" icon>
+          {i18next.t("All done!")}
+          <Header.Subheader>
+            {i18next.t("You've caught up with all open requests.")}
+          </Header.Subheader>
+        </Header>
+      </>
+    );
+  };
+
+  const NoResults = () => {
+    return (
+      <>
+        <Header icon>
+          <Icon name="search" />
+          {i18next.t("No requests found!")}
+        </Header>
+        {queryString && (
+          <Button
+            primary
+            onClick={() => props.updateQueryState(elementsToReset)}
+          >
+            {i18next.t("Reset search")}
+          </Button>
+        )}
+      </>
+    );
+  };
+
+  const allRequestsDone = is_open && !queryString;
+  return (
+    <>
+      <Segment placeholder textAlign="center">
+        {allRequestsDone ? <AllDone /> : <NoResults />}
+      </Segment>
+    </>
+  );
+};
+
+export const CommunityRequestsEmptyResultsWithState = withState(
+  CommunityRequestsEmptyResults
+);
+
+export const defaultComponents = {
```

Comment:
Please remove all the `export` above and you can also avoid to namespace them with `Community...`

```suggestion
export const defaultComponents = (appId) => {
  return {
      buildUID('SearchApp.facets', '', appId): RequestsFacets,
      ...
  }
```